pytype/tools/analyze_project/config.py

```diff
@@ -62,7 +62,9 @@
 
 # The missing fields will be filled in by generate_sample_config_or_die.
 _PYTYPE_SINGLE_ITEMS = {
-    'disable': Item(None, 'pyi-error', ArgInfo('--disable', ','.join), None),
+    'disable': Item(
+        None, 'pyi-error', ArgInfo('--disable', ','.join),
+        'Comma or space separated list of error names to ignore.'),
```

Comment:
For this to work, you'll also need to change the implementation of `generate_sample_config_or_die` below to not overwrite the comment if it is non-`None`.

Reply:
Good catch! Updated `generate_sample_config_or_die`.